src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -306,8 +223,19 @@ function setupWithdrawalAccount(params) {
                 setFreePlanVerifiedBankAccountID(responseACHData.bankAccountID);
             }
 
-            if (currentStep === CONST.BANK_ACCOUNT.STEP.REQUESTOR) {
-                checkDataAndMaybeStayOnRequestorStep(responseACHData, getNextStep(updatedACHData));
+            const nextStep = getNextStep(updatedACHData);
+
+            // Some steps require several substeps based on some data.
+            // This logic is done php side: when we need to display a specific step with specific data, php adds them in the response in nextStepValues
+            // Example 1: When forcing manual step after adding Chase bank account via Plaid, so we can ask for the real numbers instead of the plaid substitutes
```

Comment:
Just curious but when this happens are we going to show any sort of message so the user understands that it is not enough to only connect with Chase via Plaid? Like we ask you:

- Plaid or Manual
- Chooses Plaid and goes through the flow
- Gets brought to Manual flow with some explanation ?

Reply:
I don't think we need to, as it's kind of hard to explain without confusing the user more.

Reply:
Ok cool. It's pretty edge case so I agree. Putting myself in the user's shoes... if I was asked to input some account numbers manually I'd just do it to get through the flow. Thanks!